### PR TITLE
en: update docs in get started

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Update docs in Get Started to deploy TiDB with kind/minikube/GKE

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v1.1 changes for now)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:  https://github.com/pingcap/docs-tidb-operator/pull/13
- Other reference link(s):

### CODE REVIEW

File: en/deploy-tidb-from-kubernetes-gke.md [hunk 1]
```diff
@@ -86,121 +86,21 @@ kubectl get nodes
...
 If you see `Ready` for all nodes, congratulations! You've setup your first Kubernetes cluster.
 
-## Install Helm
+## Deploy TiDB Operator and TiDB cluster
 
-Helm is the package manager for Kubernetes, and is what allows us to install all of the distributed components of TiDB in a single step. Helm requires both a server-side and a client-side component to be installed.
+1. Install Helm and add the Helm chart repository maintained by PingCAP. For details, refer to [Use Helm](tidb-toolkit.md#use-helm)
 
-Install `helm`:
+2. Deploy TiDB Operator. For details, refer to [Install TiDB Operator](deploy-tidb-operator.md#install-tidb-operator).
```
Comment (yikeke) [inline en/deploy-tidb-from-kubernetes-gke.md]:
```suggestion
2. [Deploy TiDB Operator](deploy-tidb-operator.md#install-tidb-operator).
```

File: en/deploy-tidb-from-kubernetes-gke.md [hunk 2]
```diff
@@ -239,7 +139,7 @@ Try out a MySQL command inside your MySQL terminal:
 select tidb_version();
 ```
 
-If you did not specify a password in helm, set one now:
+If you did not specify a password in the process of installation, set one now:
```
Comment (yikeke) [inline en/deploy-tidb-from-kubernetes-gke.md]:
```suggestion
If you did not specify a password during installation, set one now:
```

File: en/deploy-tidb-from-kubernetes-minikube.md [hunk 3]
```diff
@@ -64,109 +64,11 @@ kubectl cluster-info
 
 ## Install TiDB Operator and run a TiDB cluster
 
-### Install helm
+1. Install Helm and add the Helm chart repository maintained by PingCAP. For details, refer to [Use Helm](tidb-toolkit.md#use-helm).
 
-Helm is the package manager for Kubernetes and is what allows us to install all of the distributed components of TiDB in a single step. Helm requires both a server-side and a client-side component to be installed.
+2. Deploy TiDB Operator. For details, refer to [Install TiDB Operator](deploy-tidb-operator.md#install-tidb-operator).
```
Comment (yikeke) [inline en/deploy-tidb-from-kubernetes-minikube.md]: ditto

File: en/deploy-tidb-from-kubernetes-minikube.md [hunk 4]
```diff
@@ -220,17 +122,23 @@ To monitor the status of the TiDB cluster, take the following steps.
 
 ### Delete TiDB cluster
 
-Use the following commands to delete the demo cluster:
+1. To delete the local cluster, refer to [Destroy TiDB Clusters in Kubernetes](destroy-a-tidb-cluster.md).
 
-```shell
-helm delete --purge demo
+2. Update the reclaim policy of PVs used by demo to `Delete`:
```
Comment (yikeke) [inline en/deploy-tidb-from-kubernetes-minikube.md]:
```suggestion
2. Update the reclaim policy of PVs used by the demo cluster to `Delete`:
```

**Comment by ran-huang** [comment]
@yikeke Comments addressed. PTAL~ Thanks!

**Pull request merged**